estuary/api/v1.py

```diff
@@ -276,7 +276,16 @@ def get_siblings(resource, uid):
         serialized_node['resource_type'] = inflated_node.__label__
         serialized_results.append(serialized_node)
 
-    return jsonify(serialized_results)
+    header = create_header(item.__label__, item.id_, item_story_flow, reverse)
```

Comment:
I think this will fail when the item is a DistGitCommit since it will have a `hash` and not `id`.